packages/jest-cli/src/TestRunner.js

```diff
@@ -379,6 +379,10 @@ class TestRunner {
   _setupReporters() {
     const config = this._config;
 
+    if (config.reporters) {
+      return this._addCustomReporters(config.reporters);
```

Comment:
Ooopsie! Let me change this. It was added there for testing purposes. Thanks for pointing it out.